.github/workflows/ci.yml

```diff
@@ -168,3 +168,20 @@ jobs:
         run: devcontainer build --image-name devcontainer-fdo-rs .
       - name: Test building in devcontainer
         run: docker run --rm -v `pwd`:/code:z --workdir /code --user root devcontainer-fdo-rs cargo build --verbose
+
+  local_packaging:
+    name: Test local packaging
+    runs-on: ubuntu-latest
```

Comment:
Should that not be Fedora or UBI as I'm not sure dnf works on ubuntu?

Reply:
Or is it running a Fedora container on Ubuntu?

Reply:
fedora container on ubuntu, see next line